Formula/ki.rb

```diff
@@ -0,0 +1,24 @@
+class Ki < Formula
+  desc "Kotlin Language Interactive Shell"
+  homepage "https://github.com/Kotlin/kotlin-interactive-shell"
+  url "https://github.com/Kotlin/kotlin-interactive-shell/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "cb9bd4faad8bbe4a87e3c8cc2ed351bd55304d991eb438bda98aa4f593e1ecba"
+  license "Apache-2.0"
+  head "https://github.com/Kotlin/kotlin-interactive-shell.git", branch: "main"
+
+  depends_on "maven" => :build
+  depends_on "openjdk"
+
+  def install
+    system "mvn", "-DskipTests", "package"
+    jar_name = Dir["ki-shell-*.jar", base: "lib"].first
+    libexec.install "lib/#{jar_name}"
+    bin.write_jar_script libexec/jar_name, "ki"
+  end
+
+  test do
+    output = shell_output("echo ':q' | #{bin}/ki")
```

Comment:
I think this could be:
```suggestion
    output = pipe_output(":q", "#{bin}/ki")
```

Reply:
I tried this and it looks like the `#pipe_output` is more for sending a file rather than a character.

```
12:00 $ brew test ki
==> Testing ki
==> :q
Error: ki: failed
An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - :q
/usr/local/Homebrew/Library/Homebrew/formula_assertions.rb:77:in `popen'
/usr/local/Homebrew/Library/Homebrew/formula_assertions.rb:77:in `pipe_output'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/ki.rb:20:in `block in <class:Ki>'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1937:in `block (3 levels) in run_test'
/usr/local/Homebrew/Library/Homebrew/utils.rb:557:in `with_env'
...
```

Reply:
oh, I think both are supposed to be commands. so it would be 
```suggestion
    output = pipe_output("echo ':q'", "#{bin}/ki")
```

Reply:
Oh! I had the [args backwards](https://www.rubydoc.info/github/Homebrew/brew/Homebrew%2FAssertions:pipe_output). `pipe_output("#{bin}/ki", ":q")` did the trick.

Thanks for the tip!